src/web/backend/ldap/models.py

```diff
@@ -157,6 +187,34 @@ def create(self):
         # TODO: better move to celery, because takes time
         self.create_teams()
 
+        # create group folder
```

Comment:
Ditto - could you please make `create` more highlevel?

Reply:
done, same